admin/class-yoast-form.php

```diff
@@ -863,8 +863,8 @@ public function index_switch( $var, $label, $help = '', $attr = [] ) {
 			$index_switch_values,
 			sprintf(
 				/* translators: %s expands to an indexable object's name, like a post type or taxonomy */
-				esc_html__( 'Show %s in search results?', 'wordpress-seo' ),
-				esc_html( $label )
+				__( 'Show %s in search results?', 'wordpress-seo' ),
```

Comment:
Please revert `esc_html__`. I think that it doesn't affect anything because %s is parameter which should be escaped separately.